sharktank/sharktank/kernels/pooling_nchw_sum.py

```diff
@@ -19,46 +19,56 @@
 class pooling_nchw_sum(CustomOp):
     """Generic pooling sum."""
 
-    signature = "pooling_nchw_sum(Tensor a, Tensor b, int[] weights_size, int[] strides, int[] padding, int[] dilations) -> (Tensor)"
+    signature = "pooling_nchw_sum(Tensor a, int[] weights_size, int[] strides, int[] padding, int[] dilations) -> (Tensor)"
 
     def select(self, ksel: KernelSelection):
-        input_desc = ksel.arg_tensor(0)  # Shape [b, ] m, k
-        inputs_pad_desc = ksel.arg_tensor(1)
-        weights_size_desc = ksel.attr_list_int(2)  # Shape [2]
-        strides_desc = ksel.attr_list_int(3)  # Shape [2]
-        padding_desc = ksel.attr_list_int(4)  # Shape [2]
-        dilations_desc = ksel.attr_list_int(5)  # Shape [2]
+        inputs_pad_desc = ksel.arg_tensor(0)
```

Comment:
At this point, I think these are just "inputs" and we define (in op comments) that it operates on explicitly padded values